scripts/dev-bootstrap

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+set -eu
+
+function run() {
+    echo "${@}"
+    "${@}"
+}
+
+for LANG_PAIR in \
+    arpeng \
+    crkeng \
+    cwdeng \
+    srseng \
```

Comment:
If you _really_ feel like it, I'm sure we could infer these lists from what files are called `*-manage.py` then use some [expansion magic](https://www.gnu.org/software/bash/manual/html_node/Shell-Parameter-Expansion.html) to get rid of the `-manage.py` stuff.

But I think this script will be so seldom used that it's not worth the confusing bashisms!

Reply:
Oh, since all the script does is call the manage scripts it wouldn’t even need expansions. That said, as more stuff gets supported for these languages, I expect I will need to replace this script with something more intelligent before we get around to adding another language.